src/Controller/PdfExportController.php

```diff
@@ -69,8 +69,8 @@ function () use ($phpWordObject) {
      */
     public function render_images($string)
     {
-        $pattern = '/\!\[([^\]]*)\]\(((?:https:\/\/|\/)[^\)]+)\)/';
-        $result = preg_replace($pattern, '$1 <img src = "$2">', $string);
+        $pattern = '/\!\[([^\]]*)\]\(((?:(?:https?|):\/\/|\/)[^\)]+)\)/';
```

Comment:
What was the intention of this change?  Do we really want to allow `://something-here` to be valid?  Or was the intention to just add the `?` after the `s` to allow `http`?

Reply:
Or was the pattern supposed to change at all?  Just the replacement (removing $1)?

Reply:
Testers have found few urls that used pattern ![alt text](//something-here). And http was not allowing in previous pattern. So for allow http and pattern similar like "://something-here." I updated pattern. At the same time I removed $1 as it was showing first bracket text along with image.